gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/JdbcAccountAccessTokenRepository.java

```diff
@@ -39,16 +41,22 @@
 public class JdbcAccountAccessTokenRepository extends AbstractJdbcRepository implements AccountAccessTokenRepository {
 
     private final SpringAccountAccessTokenRepository accessTokenRepository;
+    private final OrganizationUserRepository organizationUserRepository;
```

Comment:
**TODO**: please do no inject repository into another repository. The search user should be done once in service layer not duplicate into the JDBC & Mongo repository layer